ui_framework/components/menu/_menu.scss

```diff
@@ -1,12 +1,26 @@
+/**
+ * padding-left: 0 and list-style: none rules
+ * are meant for ul.kuiMenu and li.kuiMenuItem
+ * elements.
+ *
+ * kuiMenu and kuiMenuItems can classify both
+ * nonsemantic divs as well as lists (ul, li).
+ */
+.kuiMenu {
+  padding-left: 0;
+}
+
 .kuiMenu--contained {
   border: $globalBorderThin;
+  padding-left: 0;
```

Comment:
This isn't necessary because `kuiMenu--contained` should only be applied to elements that also have `kuiMenu` applied to it.